bin/_util.js

```diff
@@ -344,14 +344,18 @@ module.exports = {
      * @returns Variable
      */
     retrieveVariable(variableId) {
-      return Object.values(this._jsonData.variables).find((variable) => {
-        return variable.id === variableId;
-      });
+      const retrieved = Object.values(this._jsonData.variables).find(
+        (variable) => {
+          return variable.id === variableId;
+        },
+      );
+      return retrieved;
     }
   },
 
   FigmaVariable: class {
-    static TIER_1_PREFIX = 'Render/';
+    static TIER_1_PREFIX = 'render/';
+    static VARIABLE_ALIAS = 'VARIABLE_ALIAS';
```

Comment:
Note: a few of these constants will migrate to configuration once we have enough use cases to do so